src/main/java/redis/clients/jedis/JedisShardInfo.java

```diff
@@ -31,7 +32,7 @@ public int getPort() {
   public JedisShardInfo(String host) {
     super(Sharded.DEFAULT_WEIGHT);
```

Comment:
Same here. If we delegate to another constructor, we will thrown a `InvalidURIException` instead of setting the host string with the default port.